hydromt_wflow/workflows/waterbodies.py

```diff
@@ -632,7 +632,7 @@ def lakeattrs(
                 else:
                     logger.warning(
                         f"Rating data not available for lake {id}. \
-Using default Modified Puls Approach"
+Using default Modified Pulse Approach"
```

Comment:
```suggestion
Using default Modified Puls Approach"
```
It was good before :) (scientific term for the method name)